### PR TITLE
Test Case 5 : Patient Cover Sheet

### PR DESCRIPTION
**Pull request by sherrif10**
https://github.com/IsantePlus/isanteplus-qaframework/issues/18

### CODE REVIEW

**Comment by sherrif10** [comment]
Patient cover sheet Test case  is the same as patient summary

File: src/features/isanteplus/patientSummary.feature [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Feature: Patient Summary
+
+ Background:
+    Given system user logs into Isanteplus application and goes to the Home page
+
+  @patientSummary
+  Scenario: Patient Summary widget
+  When  system loads Patient dashboardPage
```
Comment (mozzy11) [inline src/features/isanteplus/patientSummary.feature:8]: can you follow the work flow desribed 


    1. Search for a patient (Use case #4.1 or 4.2)
    2. Select ‘‘Patient summary’ on the right hand side menu

**Comment by sherrif10** [comment]
Actually this should be patient cover sheet

**Comment by sherrif10** [comment]
Hey @mozzy11  My bad this is supposed to be a patient cover sheet test. kindly review

**Comment by mozzy11** [comment]
@sherrif10 , did you see the Build failures ??

**Comment by sherrif10** [comment]
Wondering why this is passing locally and failing after push, Am still debugging it

**Comment by sherrif10** [comment]
Hey @mozzy11  i think we will need to settle running servers. The problem why this test kept failing is because the data used locally forexample a name is not the same data being run on ci server.

**Comment by sherrif10** [comment]
Hello @mozzy11  is this worth merge.

**Comment by mozzy11** [comment]
Thanks  @sherrif10

**Comment by mozzy11** [comment]
Just one minor comment above

**Comment by sherrif10** [comment]
Which comment ? i seem to have addressed all the comments

File: src/features/isanteplus/patientCoversheet.feature [hunk 2]
```diff
@@ -0,0 +1,12 @@
+Feature: Patient Summary
+
+ Background:
+    Given system user logs into Isanteplus application and goes to the Home page
+
+  @patientCoverSheet
+  Scenario: Patient Cover sheet
+  When Search for and select Patient "<patientName>"
+  Then  Selected patient’s ‘Cover Sheet’ will be displayed with the following
```
Comment (mozzy11) [inline src/features/isanteplus/patientCoversheet.feature:9]: make this more meaning full.
Atleast say ``Selected patient’s ‘Cover Sheet’ will be displayed with all the right details`

Comment (sherrif10) [reply src/features/isanteplus/patientCoversheet.feature]: Resolved thnks

File: src/features/isanteplus/patientCoversheet.feature [hunk 3]
```diff
@@ -0,0 +1,12 @@
+Feature: Patient Summary
```
Comment (mozzy11) [inline src/features/isanteplus/patientCoversheet.feature:1]: is this Patient Summary or Patient cover sheet??

Comment (sherrif10) [reply src/features/isanteplus/patientCoversheet.feature]: Sorry Patient cover sheet

Comment (sherrif10) [reply src/features/isanteplus/patientCoversheet.feature]: Resolved thanks

**Comment by sherrif10** [comment]
Hello @mozzy11  kindly review thnks

**Comment by mozzy11** [comment]
Thanks @sherrif10

**Pull request merged**